src/math/StakeMath.sol

```diff
@@ -50,9 +50,9 @@ abstract contract StakeMath is MultiplierPointMath {
         returns (uint256 _deltaMpTotal, uint256 _deltaMpMax, uint256 _newLockEnd)
     {
         uint256 newBalance = _balance + _increasedAmount;
-        if (newBalance < MIN_BALANCE) {
+        /*if (newBalance < MIN_BALANCE) {
             revert StakeMath__StakeIsTooLow();
-        }
+        }*/
```

Comment:
I don't think there's a reason to keep commented out code in the code base.
Then let's just remove this.